src/utils.ts

```diff
@@ -75,4 +77,12 @@ export function stripLeadingSlash(value: string): string {
   } else {
     return value;
   }
-}
\ No newline at end of file
+}
+
+
+export class ZoneScheduler extends QueueScheduler {
+  constructor(_zone: Zone) {
+    super();
+    this.schedule = <(work: any) => Subscription>_zone.wrap(this.schedule, 'ZoneScheduler');
```

Comment:
What about
- overriding the method ?
- creating a decorator that can wrap any scheduler ?